===================================================================
RCS file: /cvs/dm/dmraid/lib/device/scsi.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- dmraid/lib/device/scsi.c	2008/09/19 14:31:11	1.3
+++ dmraid/lib/device/scsi.c	2008/10/31 12:22:35	1.4
@@ -110,7 +110,7 @@
 		}
 
 		ret = ret &&
-		     (di->serial = dbg_strdup((char *) &response[p->start + 1]));
+		      (di->serial = dbg_strdup(remove_white_space (lc, (char *) &response[p->start + 1], serial_len)));
 	}
 
 	dbg_free(response);
